pandas/tests/resample/conftest.py

```diff
@@ -126,7 +126,7 @@ def series(index, _series_name, _static_values):
 
 
 @pytest.fixture
-def empty_series(series):
+def empty_series_dti(series):
```

Comment:
Trying to avoid naming conflicts with the top level `empty_series` fixture, which I think is more general; would they not conflict if named the same?

Reply:
see #24886, @jbrockmendel removed empty series fixture since it is a singleton and inlined instead. adding a top-level empty_series fixture is inconsistent with the series tests.

The fixture here is a composable fixture and more generic and IMO should not be renamed.

Does the fixture added in this PR need to be in the top level conftest or could it be just added to pandas/tests/io/json/test_pandas.py as a module level fixture?